src/FOM/UserBundle/Resources/public/css/userlist.datatables.css

```diff
@@ -0,0 +1,86 @@
+.dataTables_filter label input {
```

Comment:
* This is a global declaration for DataTables and not only for users, but file is named as `userlist.datatables.css`
* A lot of `!important` isn't good.
* `.tdsmall`, `.dataTable` selector definitions as global defintion isn't good
* Please change it to SCSS with variable definitions